### PR TITLE
Add comments to manoeuvre faults

### PR DESCRIPTION
**Pull request by ali-campbell**
Generate typescript
Increment patch version

### CODE REVIEW

File: mes-test-schema/categories/B/index.d.ts [hunk 1]
```diff
@@ -37,14 +37,14 @@ export type ManoeuvreIndicator = boolean;
...
- */
-export type DrivingFaultCount = number;
 /**
  * Comments recorded against a fault
  */
 export type FaultComments = string;
+/**
+ * The count of the number of driving faults recorded against a test element
+ */
+export type DrivingFaultCount = number;
```
Comment (szabolcs-keresztes) [inline mes-test-schema/categories/B/index.d.ts]: Why are we creating a custom type which is basically a number?

Comment (johnmorleybjss) [reply mes-test-schema/categories/B/index.d.ts]: Personally, I'm a fan of these type aliases. I would much rather be passing around a type that indicates the intention of a given number around the codebase than a raw number. They behave in exactly the same way, the one downside being you need an extra import.

Comment (szabolcs-keresztes) [reply mes-test-schema/categories/B/index.d.ts]: The reason why I like the simple types is that you don't need to learn extra types. For example `DrivingFaultCount` could be a more complex type, but everybody knows that a `number` is a number. And if you pick the right variable names for function declarations and such you'll easily get what does that `number` represent.
Also, there is no need for an extra import :)

Comment (johnmorleybjss) [reply mes-test-schema/categories/B/index.d.ts]: Personal preference at the end of the day, both ways have pros/cons.

**Pull request merged**